index.js

```diff
@@ -55,6 +56,7 @@ var anymatch = function(criteria, value, returnIndex, startIndex, endIndex) {
     if (path.sep === '\\' && typeof string === 'string') {
       altString = string.split('\\').join('/');
```

Comment:
fwiw, I have [benchmarks in normalize-path](https://github.com/jonschlinkert/normalize-path/blob/master/benchmark/code/split.js). I used the approach that IMHO is the most performant.